src/common/auth/token_storage.ts

```diff
@@ -51,12 +58,11 @@ export function enableWrite() {
     saveTokens(tokenCache.tokens);
   }
 }
+
 export function loadTokens() {
   if (tokenCache.tokens === undefined) {
     try {
-      // Delete the old token cache.
-      delete storage.tokens;
```

Comment:
The delete is missing in the new version. Is this on purpose?

Reply:
Yes, this was 6 years ago, I assume it was called tokens and afterwards hassTokens. So not needed anymore.